templates/base.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
I prioritize to follow app hierarchy for every-single app templates folder:
https://i.imgur.com/5t3zPpU.png